src/DataTable/DataTable.scss

```diff
@@ -91,7 +91,7 @@ $data-table-footer-position: center !default;
   justify-content: space-between;
   padding: $data-table-padding-x $data-table-padding-y;
 
-  .pgn__data-table-footer-row-status {
+  .pgn__data-table-footer-left {
     align-self: $data-table-footer-position;
   }
```

Comment:
Could do `align-items: center` to align all child to center by default, removing the need to name or style `.pgn__data-table-footer-left`